openedx/features/personalized_learner_schedules/call_to_action.py

```diff
@@ -96,13 +100,43 @@ def _log_past_due_warning(name):
     @staticmethod
     def _make_reset_deadlines_cta(xblock):
         from lms.urls import RESET_COURSE_DEADLINES_NAME
-        return {
+        request = get_current_request()
+        is_learning_mfe = request and is_request_from_learning_mfe(request)
+        course_key = xblock.scope_ids.usage_id.context_key
+
+        cta_data = {
             'link': reverse(RESET_COURSE_DEADLINES_NAME),
             'link_name': _('Shift due dates'),
             'form_values': {
-                'course_id': xblock.scope_ids.usage_id.context_key,
+                'course_id': course_key,
             },
             'description': _('To participate in this assignment, the suggested schedule for your course needs '
                              'updating. Don’t worry, we’ll shift all the due dates for you and you won’t lose '
                              'any of your progress.'),
+
         }
+
+        if is_learning_mfe:
```

Comment:
You'll also need to take out the early exit for `is_learning_mfe` in the calling method.